.github/workflows/iroha2-dev-pr-wasm.yaml

```diff
@@ -33,4 +33,4 @@ jobs:
         run: cargo install --path tools/wasm_test_runner
       - name: Run smart contract tests on WebAssembly VM
         working-directory: smart_contract
-        run: mold --run cargo test --tests --target wasm32-unknown-unknown --no-fail-fast --quiet
+        run: mold --run cargo test --release --tests --target wasm32-unknown-unknown --no-fail-fast --quiet
```

Comment:
why would you do this?

Reply:
Oh, yes, so for some weird reason now `debug` compilation gives this:

```
Error: WebAssembly translation error

Caused by:
    Invalid input WebAssembly code at offset 2981644: too many locals: locals exceed maximum
```

This seems to be a [known bug](https://github.com/rustwasm/wasm-pack/issues/981) and using `release` [helps](https://github.com/xJonathanLEI/starknet-rs/pull/301#issuecomment-1519113607)